### DISCUSSION TITLE
Cannot log into Codat

### DISCUSSION THREAD

**Discussion by duebbert** [codatio/codat-docs#574] [Q&A]
Hi,

I haven't used Codat for months but still have a login. When I try to log in, it logs me in successfully but then keeps getting stuck on "Select a client to launch" with nothing to select:
![image](https://github.com/codatio/codat-docs/assets/2813849/821abdde-88fd-4044-94f3-69aa26bb0557)

Can somebody from support please look at this?

Thanks

**Comment by mcclowes** [comment]
Hi @duebbert can you please contact support? You can use [this link](https://codat.zendesk.com/hc/en-gb/requests/new) or email support@codat.io
